src/components/Attachments/AttachmentView/index.js

```diff
@@ -109,6 +109,16 @@ function AttachmentView({
         updateCurrentlyPlayingURL(isVideo ? source : null);
     }, [isFocused, isVideo, source, updateCurrentlyPlayingURL, file, isUsedInAttachmentModal]);
 
+    useEffect(
+        () => () => {
+            if (!isVideo) {
+                return;
+            }
+            currentVideoPlayerRef.current = null;
```

Comment:
Can you add a comment explaining why we need this? Here or above `useEffect` but it would be good so people understands it

Reply:
done, thanks @pecanoro